tests/integration/frameworks/test_catboost_impl.py

```diff
@@ -0,0 +1,81 @@
+import typing as t
```

Comment:
I don't think we need this file any more right? The general framework integration tests from sauyon should be more than sufficient enough